README.md

```diff
@@ -675,6 +675,18 @@ updated to reflect that change. To fix this, redeploy the lambdas (`make
 package`) in the affected deployment.
 
 
+##  Cannot update googleSubjectId with SAM
+
+If you have destroyed your deployment and are rebuilding it, it's possible that
+SAM will not allow the Google service account email to be registered again
+because the service account's email is the same in the current and previous 
+incarnation of the deployment, while the service account's uniqueID is
+different. SAM does not support this.
+
+Errors will be visible during the `make sam` step of the deployment process. To 
```

Comment:
.. or, since the heading includes it, refer to the heading.

Reply:
I changed the heading of this section from  `Cannot update googleSubjectId with SAM` to ` Unable to re-register service account with SAM` because our internal warning message does not omit a message with `googleSubjectID`, that entity is only referenced within the trace back from SAM.

I added a reference to what warning message to look for 
```
A warning message stating that `SAM does not allow re-registration of service 
account emails` will be visible during the `make sam` step of the deployment
process.
```